packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -226,6 +227,8 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
             versions: protocolVersions,
             nonce: uuid(),
             epoch: epochTracker.fluidEpoch,
+            relayUserAgent: client.details.environment !== undefined ?
+                `${client.details.environment};driverVersion:${pkgVersion}` : `driverVersion:${pkgVersion}`,
```

Comment:
I'm not sure if it works everywhere same as it works in Node, but [client.details.environment, `driverVersion:${pkgVersion}`].join("; ") seems to give what we want even if using undefined, and it feels shorter way to express what we want to do (though less efficient at execution :)).